pkg/composer/parse.go

```diff
@@ -33,3 +36,25 @@ func Parse(r io.Reader) ([]types.Library, error) {
 	}
 	return libs, nil
 }
+
+func ParseWordPress(r io.Reader) ([]types.Library, error) {
+	var libs []types.Library
+	// If wordpress file, open file and
+	// find line with content
+	// $wp_version = '<WORDPRESS_VERSION>';
+	scanner := bufio.NewScanner(r)
+	for scanner.Scan() {
+		line := strings.TrimSpace(scanner.Text())
+		if !strings.HasPrefix(line, "$wp_version") {
+			continue
+		}
+		wpVersionRegex, _ := regexp.Compile("'(.*?)'")
+		version := strings.Trim(wpVersionRegex.FindString(line), "'")
+		libs = append(libs, types.Library{
+			Name:    "wordpress",
+			Version: version,
+			License: "",
+		})
```

Comment:
Is it possible that multiple versions are defined? If no, we should break the loop.